### DISCUSSION TITLE
Using the Verbose Name for col name

### DISCUSSION THREAD

**Discussion by VaZark** [django-import-export/django-import-export#1529] [Q&A]
I just added the lib to my project and needed to the customise the col name.

I found [here](https://django-import-export.readthedocs.io/en/stable/getting_started.html#declaring-fields) that I can rename the col names manually, which is what I proceeded to do. 

However, is there anyway that the column names are automatically picked up from the `verbose_name` of a field and falls backs to the field name when it does not exist?  It helps to ensure that name remains in-step with any changes in the model.

**Comment by matthewhegarty** [comment]
Hi - there is no support for this at present.  We would consider a PR if you felt like adding this, or feel free to open an issue.  Please refer to the [contribution guidelines](https://github.com/django-import-export/django-import-export/blob/main/CONTRIBUTING.md) before starting.
